dist/types/lib/OnyxCache.d.ts.map

```diff
@@ -0,0 +1 @@
+{"version":3,"file":"OnyxCache.d.ts","sourceRoot":"","sources":["../../../lib/OnyxCache.js"],"names":[],"mappings":";AAgMA,kCAAiC;AA1LjC;;;GAGG;AACH;IAEQ;;;;OAIG;IACH,oBAA4B;IAE5B;;;;OAIG;IACH,mBAA2B;IAE3B;;;;OAIG;IACH,mBAAoB;IAEpB;;;;OAIG;IACH,wBAAyB;IAW7B;;;OAGG;IACH,cAFa,MAAM,EAAE,CAIpB;IAED;;;;OAIG;IACH,cAHW,MAAM,OAMhB;IAED;;;;OAIG;IACH,oBAHW,MAAM,GACJ,OAAO,CAInB;IAED;;;;OAIG;IACH,YAFW,MAAM,QAIhB;IAED;;;;;;OAMG;IACH,SAJW,MAAM,mBAUhB;IAED;;;OAGG;IACH,UAFW,MAAM,QAIhB;IAED;;;OAGG;IACH,YAFW,OAAO,MAAM,MAAI,QAS3B;IAED;;;;OAIG;IACH,yBAHW,MAAM,OAKhB;IAED;;;;;;;OAOG;IACH,4BAHW,MAAM,cAKhB;IAED;;;;;;;OAOG;IACH,2BAJW,MAAM,uCAUhB;IAED;;;;OAIG;IACH,0BAIC;IAED;;OAEG;IACH,oCASC;IAED;;;OAGG;IACH,0BAFW,MAAM,QAIhB;IADG,0BAA8B;CAErC"}
```

Comment:
Do we need these map files?

Reply:
I kind of feel like we should always just be creating unmodified source code with these bundles. Anything that imports them would be doing it's own bundling/minifying, so having the source code unmodified would help development, right?

Reply:
It's a bit more tricky in our case, because the bundler will have to be aware that Onyx source has `.web` and `.native` files 

See how React has `react.production.min.js` - they already provide a minified file
Even the development file comes from a pre-bundled location - `./cjs/...` 

Every library should be bundled, this relieves the consumer project form any quirks the library might otherwise require

---

The only thing helpful so far for me using unmodified Onyx source was that I can directly tweak it inside `node_modules` to experiment locally
(But I guess that's huge 😅)

Maybe we can experiment a bit and create a `.development.web.js` and `development.native.js` files, it should be as easy as setting `mode: development` 
And it should also be possible to do this:

```json
{
  "exports": {
    ".": {
       "development": "./dist/index.development.native.js"
       "default": "./dist/index.native.js"
     },
    "./web":  {
       "development": "./dist/index.development.web.js"
       "default": "./dist/index.web.js"
     },
  }
}
```